content/install_maintain/installation/installing-cli.md

```diff
@@ -66,15 +63,15 @@ The Python setup wizard prompts you to choose to install pip. If you choose not
 
 To install Cloudify CLI on Windows:
 
-1. [Download](http://cloudify.co/downloads/get_cloudify.html) the EXE installation package for Windows.
+1. [Download](http://cloudify.co/downloads) the EXE installation package for Windows.
 1. Run the installer and respond to the prompts in the installation wizard.
 
 ## Uninstalling the Cloudify CLI
 
 When you uninstall the Cloudify CLI, you only remove the CLI from the local host. This process does not change the configurations on the Cloudify Managers that you used the CLI to connect to.
 
 {{% note title="Uninstallation Note" %}}
-When you uninstall the Cloudify CLI package, the Python, pip or Virtualenv packages are not removed.
+Uninstalling the Cloudify CLI package does not remove the Python, pip or Virtualenv packages.
 {{% /note %}}
```

Comment:
\- When you uninstall the Cloudify CLI package, the Python, pip or Virtualenv packages are not removed.
\+ The Python, pip or Virtualenv packages are not removed when you remove the Cloudify CLI package.